Automattic/wp-calypso
# Jetpack Pro Dashboard: update downtime monitoring check interval UI

Related to 1204992567518369-as-1205304913870392

## Proposed Changes

* This PR updates the monitor settings modal and dashboard UI to use the monitor `check_interval` when setting the time for downtime notifications vs a deferred time. The text has been updated from `Notify me about downtime after x minutes to `Monitor my site every x minutes` to reflect this. 

<img width="471" alt="Screenshot 2023-08-23 at 8 57 55 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/96196ef5-450c-482f-9995-4e7befea8377">

## Testing Instructions



* Ensure that you have a connected site on the dashboard available.
* Apply this PR.
* Open the monitor settings modal clicking on the clock/time icon. 
* Ensure that the text in the settings modal has been updated as described in the proposed changes. 
* Ensure that the monitor intervals 1,3,5,10,15,30 minutes and 1 hour are present.
* Save each option and ensure that the payload includes a `urls` property with the site url and check interval.
* Ensure that the response from saving the updated check interval is reflected in the `urls` property in the response.
* Verify the bulk edit and Reset Notification functionality works as expected.

## Pre-merge Checklist
